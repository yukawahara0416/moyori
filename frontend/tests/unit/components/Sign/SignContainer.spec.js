import { shallowMount } from '@vue/test-utils'
import Component from '@/components/Sign/SignContainer.vue'

let wrapper

beforeEach(() => {
  wrapper = shallowMount(Component, {})
})

afterEach(() => {
  wrapper.destroy()
})

describe('v-on', () => {
  it('changeTabs', () => {
    wrapper.setMethods({ changeTabs: jest.fn() })
    wrapper.vm.$emit('changeTabs')
    expect(wrapper.emitted().changeTabs).toBeTruthy()
  })
})

describe('template', () => {
  it('snapshot', () => {
    expect(wrapper.vm.$el).toMatchSnapshot()
  })
})
