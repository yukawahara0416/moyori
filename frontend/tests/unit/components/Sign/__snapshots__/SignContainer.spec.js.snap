// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`template snapshot 1`] = `
<v-card-stub
  loaderheight="4"
  tag="div"
>
  <v-row-stub
    align="center"
    justify="center"
    no-gutter=""
    tag="div"
  >
    <v-col-stub
      tag="div"
    >
      <v-row-stub
        align="center"
        justify="center"
        no-gutter=""
        tag="div"
      >
        <v-col-stub
          cols="12"
          tag="div"
        >
          <sign-container-tabs-stub />
           
          <sign-container-tab-items-stub />
        </v-col-stub>
      </v-row-stub>
    </v-col-stub>
  </v-row-stub>
</v-card-stub>
`;
